src/ontology/templates/assays.tsv

```diff
@@ -417,7 +417,7 @@ OBI:0002968	positive/negative ion switching metabolite profiling assay			positiv
 OBI:0002969	cyclic immunoflouroescence assay				A type of microscopy assay resulting in highly multiplexed immunofluorescence imaging through sequential rounds of imaging followed by fluorophore inactivation.	PMID:27925668|PMID:29993362		Michelle Giglio|Suvarna Nadendla|Rebecca Jackson			CFDE-LINCS	https://github.com/obi-ontology/obi/issues/1243	subclass		microscopy assay														
 OBI:0002970	multiplex bead-based protein state immunoassay				A type of protein state assay in which antibodies attached to different colored beads and fluorescent reporter dye labels are used in a multiplexed mixture to assay for proteins in specific states.	url:https://www.bio-rad.com/en-us/applications-technologies/multiplex-immunoassays?ID=LUSM0E8UU		Michelle Giglio|Suvarna Nadendla|Rebecca Jackson			CFDE-LINCS	https://github.com/obi-ontology/obi/issues/1243	subclass		protein state assay														
 OBI:0002984	capture Hi-C assay			cHi-C	A Hi-C assay in which a selection step is performed on ligated fragments in order to enrich for ligation products containing specific regions of interest.	PMID:29979818	PMID:29979818 Chromosome conformation capture (3C), coupled with next-generation sequencing (Hi-C), provides a means for deciphering not only the principles underlying genome folding and architecture, but more broadly, the role 3D chromatin structure plays in gene regulation and the replication and repair of DNA.	Bonita Lam				https://github.com/obi-ontology/obi/issues/1251	subclass		Hi-C assay														
-OBI:0002985	magnetic resonance imaging assay			MRI	An imaging assay in which nuclear magnetic resonance is used to produce information about the interior structure and composition of an input material entity.	url:https://en.wikipedia.org/wiki/Magnetic_resonance_imaging		Rebecca Jackson				https://github.com/obi-ontology/obi/issues/1290	subclass		imaging assay						NMR instrument								
+OBI:0002985	magnetic resonance imaging assay			MRI	An imaging assay in which nuclear magnetic resonance is used to produce information about the interior structure and composition of an input material entity.	url:https://en.wikipedia.org/wiki/Magnetic_resonance_imaging		Rebecca Jackson|Alexander D. Bartnik, ORCID:0000-0001-9676-7377|Lucas M. Serra, ORCID:0000-0002-2104-0568|Alexander D. Diehl, ORCID:0000-0001-9990-8331|Mackenzie T. Smith, ORCID:0000-0002-9821-4132|William D. Duncac, ORCID:0000-0001-9625-1899|Lauren M. Wishnie, ORCDI:0000-0002-7245-3450				https://github.com/obi-ontology/obi/issues/1290	subclass		imaging assay						NMR instrument			 ('has role' some 'magnetic resonance imaging evaluant role')					(realizes some (concretizes some 'magnetic resonance pulse sequence protocol'))
```

Comment:
- typo "Duncac"
- typo "ORCDI"

Reply:
Thanks for catching! I corrected them in the latest commit.